spec/features/newflow/educators/educator_signup_flow_spec.rb

```diff
@@ -60,9 +60,9 @@ module Newflow
           select_educator_role('other')
           fill_in('Other (please specify)', with: 'President')
           click_on('Continue')
-          expect(page.current_path).to eq(signup_done_path)
-          click_on('Finish')
-          expect(page.current_url).to eq(external_app_url)
+          #expect(page.current_path).to eq(signup_done_path)
```

Comment:
Should this be commented out? There is another section below.

Reply:
I think this was leftover from debugging - uncommented